content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Summary from the 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)
+date: 2022-07-15T19:31:06.645Z
+priority: 2
+author: Michelle Strout and Engin Kayraklioglu
+authorimage: /img/combined-author-photos.jpg
+thumbnailimage: /img/chapel-logo.jpg
+tags:
+  - Chapel Programming Language
+  - GPU Programming
+  - High Performance Computing
+  - Open Source
+  - Arkouda (Data Analytics in Python powered by Chapel)
+---
```

Comment:
Could you please one additional tag: chapel
This will allow to tie your blog post to the platform page "Chapel" in the dev portal.